plugins/utils/neorg.nix

```diff
@@ -0,0 +1,149 @@
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}: let
+  cfg = config.plugins.neorg;
+  helpers = import ../helpers.nix {inherit lib;};
+in
+  with lib; {
+    options.plugins.neorg =
+      helpers.extraOptionsOptions
+      // {
+        enable = mkEnableOption "neorg";
+
+        package = helpers.mkPackageOption "neorg" pkgs.vimPlugins.neorg;
+
+        lazyLoading =
+          helpers.defaultNullOpts.mkBool false ''
+          '';
+
+        logger = let
+          modes = {
+            trace = {
+              hl = "Comment";
+              level = "trace";
+            };
+            debug = {
+              hl = "Comment";
+              level = "debug";
+            };
+            info = {
+              hl = "None";
+              level = "info";
+            };
+            warn = {
+              hl = "WarningMsg";
+              level = "warn";
+            };
+            error = {
+              hl = "ErrorMsg";
+              level = "error";
+            };
+            fatal = {
+              hl = "ErrorMsg";
+              level = "5";
+            };
```

Comment:
It  seemed  like level == attrName but the last one says nope....

Reply:
Maybe you could check if the defaults set has a `level` attr and act differently, but I feel it's too complicated

Reply:
Yes, I didn't want to overdo it. Sometimes, a bit more redundant but simpler code is better !